fastestimator/pipeline.py

```diff
@@ -362,6 +416,7 @@ def __call__(self, mode: str, epoch: int = 1, shuffle: Optional[bool] = None,
             raise ValueError("You cannot invoke a Pipeline's __call__ method while it already has an active loader.")
         self.ctx_mode = mode
         self.ctx_epoch = epoch
+        self.ctx_ds_id = ds_id
```

Comment:
this var should also be declared in init

Reply:
declared